lib/katello/permission_creator.rb

```diff
@@ -12,6 +12,7 @@ def define
       host_collections_permissions
       lifecycle_environment_permissions
       product_permissions
+      #repository_permissions
```

Comment:
I have these disabled at the moment, but we might consider making repository permissions distinct from product permissions. The foreman auth model does not (at least as it appears to me) support our desire to treat repos the same as products permission-wise.